cli/lib/kontena/command.rb

```diff
@@ -218,15 +218,19 @@ def run(arguments)
     end
   rescue Kontena::Errors::StandardError => ex
     raise ex if ENV['DEBUG']
-    abort(" [#{Kontena.pastel.red('error')}] #{ex.class.name} : #{ex.message}")
+    Kontena.logger.error("ERROR") { "#{ex.class.name} : #{ex.message}" }
```

Comment:
Fun fact: `Logger` can also take an `Exception` and print it out, including a backtrace and stuff:

```ruby
irb(main):007:0> begin
irb(main):008:1*   fail "something"
irb(main):009:1> rescue => exc
irb(main):010:1>   logger.error(exc)
irb(main):011:1> end
E, [2017-05-04T17:14:29.874143 #10387] ERROR -- : something (RuntimeError)
(irb):8:in `irb_binding'
...
```

Should we just be using that everywhere, instead of the hundreds of lines of hand-rolled exception-formatting that we have everywhere?